template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-FSharp/Startup.fs

```diff
@@ -8,33 +8,21 @@ open Microsoft.AspNetCore.Builder
 open Microsoft.AspNetCore.Hosting
 open Microsoft.Extensions.Configuration
 open Microsoft.Extensions.DependencyInjection
-open Microsoft.Extensions.Logging
 
 
 type Startup private () =
```

Comment:
This is more of a style issue and was already there in the original, but unless the framework relies on having both a empty constructor and a constructor that takes an IConfiguration object, you can just use F#'s primary constructor syntax:

this...
```
type Startup (configuration: IConfiguration) =
```
instead of this...
```
type Startup private () =
    new (configuration: IConfiguration) as this =
        ...
```
_note that the empty constructor is/has been private all along so I doubt it is relied on..._